base_app/src/index.html

```diff
@@ -16,27 +16,37 @@
     let url = new URL(window.location.href);
     let uiProp = url.searchParams.get("uiProp");
 
-    let webpackURI = appURI + '/js/';
     document.title = appURI;
 
     console.log("appURI : ", appURI );
     let link = document.createElement('link');
     link.setAttribute('rel', 'icon');
     link.type = 'image/x-icon';
-    link.href = appURI + '/images/favicon.ico';
+    link.href = appURI + 'assets/images/favicon.ico';
     document.head.appendChild(link);
 
+    // this condition is specifically for ng serve dev env
+    let debugPrefix = "";
+    if (window.location.port === '4200') {
+      debugPrefix = window.location.protocol + '//' + window.location.hostname + ':4100';
+      link.href = window.origin + '/bms/assets/images/favicon.ico';
+      console.log(`debugPrefix: ${debugPrefix}`);
+    }
+
     let base = document.getElementsByTagName('base')[0];
-    base['appURI'] = appURI ;
+    base['appURI'] = appURI;
     base['uiProp'] = uiProp;
-    base['propsQuery'] = appPath + (uiProp || 'ui') + '/query/props';
+    base['propsQuery'] = debugPrefix + appPath + (uiProp || 'ui') + '/query/props';
+    console.log(`base['propsQuery']: ${base['propsQuery']}`);
+
 
     if (uiProp) {
-        document.title = uiProp;
+      document.title = uiProp;
     }
 
   </script>
   <base href=appURI />
+  <link href="/bms/assets/image-overlays.css" rel="stylesheet" />
```

Comment:
Don't think `/bms/assets/image-overlays.css` will exist for generic app running in the regular docker container.
I think this should be injected by angular / webpack.

Reply:
The `/${App}/overlay-${nthOverlay}/image-overlays.css` is not known at build time.
It's an external file that has to be connected during deployment / bundling into the docker container.

Reply:
Oooooooooh that makes sense, now I see the problem